matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -870,11 +881,19 @@ var PhenoTips = (function (PhenoTips) {
                 + '<option value="saved" '+ (match.status == "saved" ? ' selected="selected"' : '') + '>' + this._SAVED + '</option>'
                 + '<option value="rejected" '+ (match.status == "rejected" ? ' selected="selected"' : '') + '>' + this._REJECTED + '</option>'
                 + '</select>';
-            var icon = (match.comment && match.comment != "") ? "fa fa-comment" : "fa fa-comment-o";
+            // comments
+            var icon = (match.comments) ? "fa fa-comments" : "fa fa-comments-o";
             td += '<span class="buttonwrapper" title="' + this._ADD_COMMENT_TITLE + '"><a class="button comment" href="#"><span class="' + icon + '"> </span></a></span>';
-            td += '<div class="xTooltip comment-container"><span class="hide-tool" title="Hide">×</span><div><textarea rows="5" cols="20"></textarea></div>'
-                +'<span class="buttonwrapper"><a class="button save-comment" data-matchid="' + match.id + '" href="#"><span class="fa fa-save"> </span>'
-                + this._SAVE_COMMENT_BUTTON_LABEL + '</a></span></div>';
+            var commentsTable = '';
+            if (match.comments) {
+                commentsTable = this._generateCommentsTable(match.comments);
+            }
+            td += '<div class="xTooltip comment-container"><span class="hide-tool" title="Hide">×</span>'
+                + '<div class="nhdialog-title">' + this._COMMENTS_TITLE + '</div>'
+                + '<p class="xHint">' + this._COMMENTS_HINT + '</p>'
```

Comment:
Didn't we decide yesterday to turn this into an (i) or (?) button?

Reply:
We were testing yesterday with Sasha and noticed that notes have this info not styled as icon, so for now for consistency with notes dialog we can leave like text for now and change it later. I made it smaller with less line space so it is not taking much space now.

Reply:
We can move text to help icons when we make all pop-ups as a separate widget later.